compiler/circle-opselector/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ set(DRIVER "driver/Driver.cpp")
 file(GLOB_RECURSE SOURCES "src/*.cpp")
 
 add_executable(circle-opselector ${DRIVER} ${SOURCES})
-target_include_directories(circle-opselector PRIVATE src)
```

Comment:
I think this part is occuring error.
This seems to have been merged incorrectly.
> target_include_directories(...pass)

pass -> src